### DISCUSSION TITLE
Hero 11 mini photo mode

### DISCUSSION THREAD

**Discussion by jerops** [gopro/labs#482] [Feature Requests]
Hi, 
Would it be possible to add photo mode to the gopro hero 11 mini?

**Comment by dnewman-gpsw** [comment]
No, Labs can't add modes, it can only tweak modes that are already there.

**Reply by jerops** [reply]
Thanks. Do you know if there is any change to get that functionality on the camera? On my team, there are many who would like to buy this camera model, but the lag of photo mode stops us.

**Reply by dnewman-gpsw** [reply]
Very unlikely. For two reason I can guess at: all video can be turned in photos, and the idea of framing a single photo without a screen is likely not a common use case.

**Reply by jerops** [reply]
Ok. Would it be possible to write a script to add that functionality or to
load the hero 11 firmware loaded?

Thanks

**Reply by dnewman-gpsw** [reply]
No, that photo mode in not there to script.  HERO11 firmware is different enough, it will not install.
